x-pack/plugins/apm/public/index.js

```diff
@@ -43,15 +45,17 @@ initTimepicker(history, store.dispatch).then(() => {
   );
 
   ReactDOM.render(
-    <Provider store={store}>
-      <Fragment>
-        <GlobalProgress />
-        <LicenseChecker />
-        <Router history={history}>
-          <Main />
-        </Router>
-      </Fragment>
-    </Provider>,
+    <I18nProvider>
```

Comment:
Is `I18nProvider` still needed?

Reply:
No, it isn't needed in this PR. 
Removed.